cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup cannoli oil
```

Comment:
Do you really want a cannoli in the cake? Change to appropriate oil name and resubmit.